Artifacts.toml

```diff
@@ -988,6 +988,28 @@ os = "linux"
     sha256 = "a2d2582098364253f601285c790e1a91893602c4963c76c59598e4671fee6ce6"
     url = "https://github.com/JuliaPackaging/Yggdrasil/releases/download/GCCBootstrap-v12.1.0+1/GCCBootstrap-i686-linux-gnu.v12.1.0.x86_64-linux-musl.unpacked.tar.gz"
 
+[["GCCBootstrap-i686-linux-gnu.v13.2.0.x86_64-linux-musl.squashfs"]]
```

Comment:
I'm not sure why these are new - I was just using the platform list in `gcc_common.jl`

Reply:
Sounds like @gbaraldi missed this platform in #323?  But now I don't understand how we got the CSL package for `i686-linux-gnu` :confused: